src/models/BaseModel.py

```diff
@@ -197,9 +242,12 @@ def load_cleaned_data():
         loaded_data = pd.read_csv(BaseModel.CLEANED_FILE_NAME, header=None)
         loaded_data = loaded_data.as_matrix()
         loaded_data = loaded_data.astype(int)
-        return loaded_data
 
-    abc.abstractmethod
+        loaded_feature_data = pd.read_csv(BaseModel.FEATURE_METADATA_FILE_NAME,
```

Comment:
by filename, kept it this way for consistency with how other files are getting written/read